flybywire-aircraft-a320-neo/en-US.locPak

```diff
@@ -16,7 +16,7 @@
       "A32NX.LOADING.TIPS.A320NEO_001": "The FlyByWire A32NX is a collaboration between hundreds of individual contributors.",
       "A32NX.LOADING.TIPS.A320NEO_002": "Did you know? The A320 can land at crosswinds up to 38 knots and tailwinds of up to 15 knots!",
       "A32NX.LOADING.TIPS.A320NEO_003": "FlyByWire is named after the fly-by-wire controls first introduced in the Airbus A320.",
-      "A32NX.LOADING.TIPS.A320NEO_004": "There are 9693 A320 family aircraft built! That's almost as much as the population of Tuvalu!",
+      "A32NX.LOADING.TIPS.A320NEO_004": "There are 9873 A320 family aircraft built! That's almost as much as the population of Tuvalu!",
```

Comment:
One could do the same thing as proposed in german above.

switch the logic to not use an single number for a constantly changing value, use a range instead.

and once passed 10k we change it to be more than 10K. This way the value is never really outdated and we don't need to constantly update production numbers.

```suggestion
      "A32NX.LOADING.TIPS.A320NEO_004": "There are almost 9873 A320 family aircraft built! That's almost as much as the population of Tuvalu!",
```

Reply:
I have to disagree; `about` would probably be a better word as `almost` signifies that the real number is approaching that value, but has not approached it.